README.md

```diff
@@ -102,6 +102,33 @@ where the currently available models are:
 - [DFKDE](anomalib/models/dfkde)
 - [GANomaly](anomalib/models/ganomaly)
 
+### Custom Dataset
+It is also possible to train on a custom dataset. To do so, `data` section in `config.yaml` is to be modified as follows:
+```yaml
+dataset:
+  name: custom
```

Comment:
Maybe we should use `format` here instead of `name`. For MVTec we also have a `format` field in addition to `name`. The way I see it, `format` determines which dataset class is used under the hood, while `name` can be anything that identifies the specific dataset that is used.